scripts/TEST.vcf

```diff
@@ -0,0 +1,500 @@
+##fileformat=VCFv4.2
+##FILTER=<ID=PASS,Description="All filters passed">
+##FILTER=<ID=Consensus,Description="Called by two or more callers">
+##FILTER=<ID=Hotspot,Description="In mutational hotspot, as defined by ????">
```

Comment:
Why did we choose to make these filters? Seems like Hotspot and Consensus are more descriptive of PASS variants than they are reasons the variant is not PASS. This thought could be totally wrong though.

Reply:
I think this was a miscommunication on my part. I think I originally envisioned it as a filter, but as Dan and I discussed and worked on hot spots, it became apparent that `INFO` was the way to go, but this info never made it to Nathan.

Reply:
Switching 'Hotspot' to INFO flag, all filters will be PASS